tests/speculos/test_eip1559.py

```diff
@@ -0,0 +1,65 @@
+from time import sleep
+
+from ethereum_client.utils import apdu_as_string, compare_screenshot, save_screenshot, PATH_IMG, parse_sign_response
+
+def test_sign_eip_1559(cmd):
+    result: list = []
+    # with bip32_path "44'/60'/0'/0/0"
+    apdu_sign_eip_1559 = apdu_as_string("e004000088058000002c8000003c80000000000000000000000002f87001018502540be4008502540be40086246139ca800094cccccccccccccccccccccccccccccccccccccccc8000c001a0e07fb8a64ea3786c9a6649e54429e2786af3ea31c6d06165346678cf8ce44f9ba00e4a0526db1e905b7164a858fd5ebd2f1759e22e6955499448bd276a6aa62830")
+
+    with cmd.send_apdu_context(apdu_sign_eip_1559, result) as ex:
+        sleep(0.5)
+        
+        if cmd.model == "nanos":
```

Comment:
There is a lot of lines to factorize here (and I think it is true for all tests):
```python
# Review transaction
compare_screenshot(cmd, f"screenshots/eip1559/{PATH_IMG[cmd.model]}/sign_eip_1559/00000.png")
cmd.client.press_and_release('right')

# Amount
compare_screenshot(cmd, f"screenshots/eip1559/{PATH_IMG[cmd.model]}/sign_eip_1559/00001.png")
cmd.client.press_and_release('right')

# Address and fees (longer for NanoS)
compare_screenshot(cmd, f"screenshots/eip1559/{PATH_IMG[cmd.model]}/sign_eip_1559/00002.png")
cmd.client.press_and_release('right')
compare_screenshot(cmd, f"screenshots/eip1559/{PATH_IMG[cmd.model]}/sign_eip_1559/00003.png")
cmd.client.press_and_release('right')
next_pic = 4
if cmd.model == "nanos":
    compare_screenshot(cmd, f"screenshots/eip1559/{PATH_IMG[cmd.model]}/sign_eip_1559/00004.png")
    cmd.client.press_and_release('right')
    compare_screenshot(cmd, f"screenshots/eip1559/{PATH_IMG[cmd.model]}/sign_eip_1559/00005.png")
    cmd.client.press_and_release('right')
    next_pic += 2

# Accept and send
compare_screenshot(cmd, f"screenshots/eip1559/{PATH_IMG[cmd.model]}/sign_eip_1559/0000{next_pic}.png")
cmd.client.press_and_release('both')
```

I agree, the comments won't be as accurate. But it's tests. Sure, they should be easily understandable, but conciseness is also a quality.

Reply:
I am afraid that by factoring too much we lose readability in the tests

Reply:
Debatable, but I get your point.